server/services/employers.js

```diff
@@ -68,18 +68,26 @@ const getSitesForUser = async (user) => {
     criteria.siteId = user.sites.map((site) => `${site}`); // cast to string
   }
 
-  return dbClient.db[collections.EMPLOYER_SITES].findDoc(criteria, {
-    order: [{ field: `siteName`, direction: 'asc' }],
-    fields: [
-      'siteId',
-      'siteName',
-      'operatorName',
-      'city',
-      'healthAuthority',
-      'postalCode',
-      'allocation',
-    ],
-  });
+  const records = (
+    await dbClient.db[collections.EMPLOYER_SITES].findDoc(criteria, {
```

Comment:
I'm confused- this is the same function as https://github.com/bcgov/hcap/pull/834/files, right? Is this PR out of date and should be closed? Or are you wanting the other changes?

Reply:
Sorry #834 was branched off this one, they're merged together now